packages/monitoring/package.json

```diff
@@ -4,10 +4,37 @@
   "version": "0.3.0",
   "author": "Google Inc",
   "description": "Stackdriver Monitoring API client for Node.js",
+  "contributors": [
```

Comment:
We don't need these hand-edits at all. The library is auto-generated. So there will be no 'contributors' and 'CONTRIBUTING'.

Please keep this file as is.